packages/fxa-auth-db-mysql/lib/db/mem.js

```diff
@@ -959,6 +959,7 @@ module.exports = function(log, error) {
       deleteByUid(uid, passwordChangeTokens);
       deleteByUid(uid, passwordForgotTokens);
       deleteByUid(uid, unverifiedTokens);
+      deleteByUid(uid, recoveryKeys);
```

Comment:
This is the only important change in this file, everything else is style changes.